pages/about/AboutContextHandler.inc.php

```diff
@@ -79,6 +79,36 @@ function submissions($args, $request) {
 
 		$templateMgr->assign( 'submissionChecklist', $context->getLocalizedSetting('submissionChecklist') );
 
+		// Get section options for this context
+		$roleDao = DAORegistry::getDAO('RoleDAO');
+		$user = $request->getUser();
+
+		$canSubmitAll = false;
+		if ($user) {
+			$canSubmitAll = $roleDao->userHasRole($context->getId(), $user->getId(), ROLE_ID_MANAGER) ||
+				$roleDao->userHasRole($context->getId(), $user->getId(), ROLE_ID_SUB_EDITOR);
+		}
+
+		$sectionDao = Application::getSectionDAO();
+		$sections = $sectionDao->getTitles($context->getId(), !$canSubmitAll);
+
+		if (count($sections) > 0) {
+			array_walk($sections, function (&$item, $sectionId) use ($sectionDao) {
+				$item = array('title' => $item);
+
+				$section = $sectionDao->getById($sectionId);
+
+				$sectionPolicy = $section ? $section->getLocalizedPolicy() : null;
+				$sectionPolicyPlainText = trim(PKPString::html2text($sectionPolicy));
+				if (strlen($sectionPolicyPlainText) > 0)
```

Comment:
Is this to avoid section policies containing e.g. `<p></p>`? Maybe it would be better to address this on the storage side rather than the presentation side. (Especially since this same code also appears in https://github.com/pkp/ojs/pull/2132.)

Reply:
This is basically the check for "is there a policy available". The code retrieves the localized policy, then converts it to plain text to determine whether the policy contains any readable text (as for example `<p></p>` isn't really a policy that needs to be displayed). So when a text is available the template variable to render the text in template accordingly. I hope this makes sense. Perhaps i should move this code to a method named like `doesSectionPolicyContainsAnyText()` to make the code more readable? What do you say?

Reply:
I think `SectionDAO` (OJS)/`SeriesDAO` (OMP)/`PKPSectionDAO` (pkp-lib) is the best candidate.

Reply:
Done via https://github.com/ViRuSTriNiTy/ojs/commit/ca35300a92d79c19b58b2947f430cee1f55de5ed